platform/ios/MGLMapView.mm

```diff
@@ -383,6 +383,40 @@ - (void)commonInit
     }];
 }
 
+- (NS_ARRAY_OF(NSDictionary *) *)featuresAt:(CGPoint)point
+{
+    // flip y for core
+    point.y = self.bounds.size.height - point.y;
```

Comment:
Like the JS API, we do the work of figuring out the tile deep down in the source. Converting to a coordinate still requires us to access the transform in the source since we still need to use the `Transform::pointToCoordinate` API to determine the `TileCoordinate` (separate type from both `LatLng` and screen `PrecisionPoint`. We also need access to the current `Transform::scale` since we need are querying for features at the resolution of the tile extent (i.e. 4096) and the scale informs the query box size in terms of the radius passed.